services/transformer/features.go

```diff
@@ -74,8 +76,8 @@ func (*noopService) Regulations() []string {
 }
 
 func (*noopService) SourceTransformerVersion() string {
-	// v0 is deprecated
-	return V1
+	// v0 is deprecated and upgrading to v2
```

Comment:
features_impl.go returns version based on `upgradedToSourceTransformV2` value received by /features response from transformer

when does this come into picture ? and what difference it makes returning v1 vs. v2 here ?

Reply:
This is used in some test cases as a placeholder for transformer service. This used to return v0 earlier but as v0 is deprecated v1 is the ideal response.
This doesn't affect the actual logic anywhere.